Makefile.am

```diff
@@ -4,6 +4,7 @@ SUBDIRS = doc include src test
 
 dist_doc_DATA = AUTHORS ChangeLog INSTALL LICENSE README.md
 EXTRA_DIST = libica.map libica.spec
+MAJOR = `echo $(VERSION) | cut -d. -f1`
```

Comment:
Should be a `:=` here to not unnecessarily spawn a sub-shell whenever MAJOR is used.

Reply:
Hi Juergen, you mean 
MAJOR := `echo $(VERSION) | cut -d. -f1`
?

Reply:
yes

Reply:
ok, done.